Makefile

```diff
@@ -46,17 +46,17 @@ server: compose-build
 server-shell: compose-build
 	@docker-compose run $(IMAGE_NAME) /bin/bash
 
-integration-shell: compose-build
+integration-shell:
 	@virtualenv -p python3 fidesops_test_dispatch; \
 		source fidesops_test_dispatch/bin/activate; \
 		python run_infrastructure.py --open_shell --datastores $(datastores)
 
-integration-env: compose-build
+integration-env:
```

Comment:
Sean also pointed out that this `compose-build` call calls docker-compose down and then docker-compose build which is why this would restore our databases when called and repopulate with the seed data script.